pkg/config/config.go

```diff
@@ -153,6 +155,7 @@ const (
 	docDBUrlKey              = "storage.docdb.url"
 	grafeasProjectIDKey      = "storage.grafeas.projectid"
 	grafeasNoteIDKey         = "storage.grafeas.noteid"
+	grafeasNoteHintKey       = "storage.grafeas.notehint"
```

Comment:
This added feature seems unrelated to the broader goal of the PR. Can we split this out into another change? (this PR looks close, so feel free to base it off this branch).